benchmarks/sqlite3_ossfuzz/ossufzz.dict

```diff
@@ -0,0 +1,282 @@
+#
```

Comment:
typo in filename ossufuzz.dict, can you recheck all new filenames or are there any others you manually created